ibm_cloud_sdk_core/__init__.py

```diff
@@ -25,6 +25,8 @@
 functions:
     datetime_to_string: Serializes a datetime to a string.
     string_to_datetime: De-serializes a string to a datetime.
+    datetime_list_to_string_list: Serializes a list of datetimes to a list of strings.
```

Comment:
I think we also need `date_to_string_list` and `string_to_date_list`, don't we?

Reply:
Not needed for the python unit tests since the example value can be used without reformatting. The main reason datetime has an issue is because the python conversion of datetime to string uses `datetime.isoformat()` and ends up with some formatting differences to the original example. Whereas dates are simple converted to string `str(date)` so the unit tests don't need an extra util function for those